### PR TITLE
Add test for use FRocksDB as a state backend

### PR DESCRIPTION
**Pull request by kornys**

### CODE REVIEW

File: src/test/java/io/streams/e2e/flink/sql/SqlJobRunnerST.java [hunk 1]
```diff
@@ -328,4 +334,225 @@ void testWrongConnectionInfo() {
...
+            @Step(value = "Verify that taskmanager logs contains 'State backend loader loads the state " +
+                "backend as EmbeddedRocksDBStateBackend'", expected = "Log message is present"),
+        },
+        labels = {
+            @Label(value = FLINK_SQL_RUNNER),
+            @Label(value = FLINK),
+        }
+    )
+    @Test
+    void testFRocksDbStateBackend() {
```
[the remark this answers is not in the document]
Comment (kornys) [reply src/test/java/io/streams/e2e/flink/sql/SqlJobRunnerST.java:368]: probably yes but I would like to do it in different PR

[the remark this answers is not in the document]
Comment (Frawless) [reply src/test/java/io/streams/e2e/flink/sql/SqlJobRunnerST.java:369]: Sure, no problem

**Pull request merged**